src/libraries/System.Text.RegularExpressions/gen/Resources/xlf/Strings.cs.xlf

```diff
@@ -194,7 +194,7 @@
       </trans-unit>
       <trans-unit id="NotSupported_NonBacktrackingAndReplacementsWithSubstitutionsOfGroups">
         <source>Regex replacements with substitutions of groups are not supported with RegexOptions.NonBacktracking.</source>
-        <target state="new">Regex replacements with substitutions of groups are not supported with RegexOptions.NonBacktracking.</target>
+        <target state="translated">Nahrazení regulárních výrazů pomocí substitucí skupin se u regexOptions.NonBacktracking nepodporuje.</target>
```

Comment:
Should've `RegexOptions.NonBacktracking` be an href so that is not localized? 

cc: @ericstj @stephentoub

Reply:
I wonder if we can just reject that localization by changing the state?  Perhaps we should add a note when doing so that `RegexOptions.NonBacktracking` should not be localized.  Let me ping @cristianosuzuki77 and @jonfortescue to see if they know.

Reply:
Yeah, you can add a comment to the source resx (which will be carried through to the xlf) just saying "do not localize this"; that's usually the best way to handle it.

Reply:
That's what I thought. I'll put up a PR to fix those. Thanks!